help/en/package/jmri/jmrit/withrottle/Protocol.shtml

```diff
@@ -7,7 +7,7 @@
 
 <!-- Style -->
 <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
-<LINK REL="stylesheet" TYPE="text/css" HREF="/css/default.css" MEDIA="screen">
+<LINK REL="stylesheet" TYPE="text/css" HREF="http://jmri.org/css/default.css" MEDIA="screen">
```

Comment:
Woops. Thanks. I’ll fix that.

From: Randall Wood [mailto:notifications@github.com]
Sent: Monday, January 18, 2016 9:32 PM
To: JMRI/JMRI <JMRI@noreply.github.com>
Cc: John Socha-Leialoha <john@socha.com>
Subject: Re: [JMRI] Expanding WiThrottle Protocol Docs (#669)


In help/en/package/jmri/jmrit/withrottle/Protocol.shtml<https://github.com/JMRI/JMRI/pull/669#discussion_r50074070>:

> @@ -7,7 +7,7 @@

>

>  <!-- Style -->

>  <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">

> -<LINK REL="stylesheet" TYPE="text/css" HREF="/css/default.css" MEDIA="screen">

> +<LINK REL="stylesheet" TYPE="text/css" HREF="http://jmri.org/css/default.css" MEDIA="screen">

Since this page should be runnable locally, this line needs to be reverted.

—
Reply to this email directly or view it on GitHub<https://github.com/JMRI/JMRI/pull/669/files#r50074070>.